driver/src/lib.rs

```diff
@@ -29,7 +29,8 @@ impl NRF24L01 {
     pub async fn open<P: AsRef<Path>>(device: P, config: Configuration) -> Result<Standby, Error> {
         // Open the serial port.
         let settings = SerialPortSettings::default();
-        let mut port = Serial::from_path(device, &settings).unwrap();
+        let port = Serial::from_path(device, &settings).unwrap();
```

Comment:
That is not going to compile on Linux, as the `mut` is still required on that platform.